modules/alias/main.tf

```diff
@@ -108,3 +108,113 @@ resource "aws_lambda_permission" "qualified_alias_triggers" {
   source_account     = try(each.value.source_account, null)
   event_source_token = try(each.value.event_source_token, null)
 }
+
+resource "aws_lambda_event_source_mapping" "no_refresh" {
```

Comment:
I think these two resources `aws_lambda_event_source_mapping` can be combined into one.

Reply:
Good call. Changes to make this a single resource have been implemented.